lightwood/analysis/nc/nc.py

```diff
@@ -466,19 +466,19 @@ def predict(self, x, nc, significance=None):
         n_test = x.shape[0]
         prediction = self.model.predict(x)
 
-        norm = np.ones(n_test)
+        norm = 1
```

Comment:
why is it scalar when before it needed to be an array?

Reply:
I don't think it _needed_ to be an array. This change forces `numpy` to broadcast it anyway. Not sure if it's faster or not, though.